### DISCUSSION TITLE
Table feature inclusion

### DISCUSSION THREAD

**Discussion by nyirongojr** [vueform/builder#27] [Ideas]
### Describe the feature

Is there a possibility of including a table component in the drag and drop. It would really assist in arranging items and presenting them in an organized manner.

### Final checks

- [X] Check existing [issues](https://github.com/vueform/builder/issues) and [ideas](https://github.com/vueform/builder/discussions/categories/ideas).
- [X] Check [roadmap](https://github.com/vueform/builder/discussions/categories/roadmap).

**Comment by adamberecz** [comment]
Related to [vueform#73](https://github.com/vueform/vueform/discussions/73)

**Comment by adamberecz** [comment]
Finally 🎉
https://vueform.com/reference/matrix-element

To check out its features search for "Matrix" or "Table" element:
https://builder.vueform.com/demo
